test/FileFetcherTest.php

```diff
@@ -50,18 +54,29 @@ public function testTimeOut()
         $fetcher2->run();
         $this->assertEquals($file_size, $fetcher2->getStateProperty('total_bytes_copied'));
         $this->assertEquals(
-            filesize("/tmp/{$this->sampleCsvSize}_mb_sample.csv"),
+            filesize("/tmp/dkan_default_content_files_s3_amazonaws_com_{$this->sampleCsvSize}_mb_sample.csv"),
             $fetcher2->getStateProperty('total_bytes_copied')
         );
         $this->assertEquals($fetcher2->getResult()->getStatus(), \Procrastinator\Result::DONE);
     }
 
+    public function testIncompatibleServer()
+    {
+        $url = "https://data.medicare.gov/api/views/42wc-33ci/rows.csv?accessType=DOWNLOAD&sorting=true";
```

Comment:
Dangerous!

Reply:
@dafeder Agreed, but what options do we have?